analyses/tmb-calculation/README.md

```diff
@@ -1,45 +1,46 @@
 # OpenPencan Tumor Mutation Burden calculation
 
-This analysis utilizes the SNV consensus MAF file, `../../data/snv-consensus-plus-hotspots.maf.tsv.gz` from [Pediatric Open Targets, OPenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) datasets to calculate Tumor Mutation Burden (TMB) for each experimental strategy (WGS, WXS, and Targeted Sequencing) tumor sample with SNV calls for all cohorts and cancer types evaluated in the project. The Tumor Mutation Burden calculation is adapted from [snv-callers module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) of the OpenPBTA-analyses, and use the SNV calls Mutect2, Strelka2, Lancet, and Vardict callers.
+This analysis utilizes the SNV consensus MAF file, `../../data/snv-consensus-plus-hotspots.maf.tsv.gz` from [Pediatric Open Targets, OPenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) datasets generated using [Kids First DRC Consensus Calling Workflow](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc-consensus-calling.md)to calculate Tumor Mutation Burden (TMB). TMB is calculated for tumor sample in each experimental strategy (WGS, WXS, and Targeted Sequencing) using SNV calls for all cohorts and cancer types evaluated in the project. The TMB calculation is adapted from [snv-callers module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) of the OpenPBTA-analyses, but uses nucleotide variants (SNVs) in the consensus file called in at least 2 of the 4 caller (Mutect2, Strelka2, Lancet, and Vardict) utilized in the consensus calling workflow and/or 1 of the 4 callers if SNVs are considered hotspots. The SNV consensus is split into two subsets of SNVs and multinucleotide variants (MNVs). The MNVs subset is split into SNVs, merged to the SNVs subset, and sample-specific redundant calls removed. The resulting merged and non-redundant SNV consensus calls togther with sample-specific BED files are utilized for TMB calculation as described below. 
```

Comment:
```suggestion
This analysis utilizes the SNV consensus MAF file, `../../data/snv-consensus-plus-hotspots.maf.tsv.gz` from [Pediatric Open Targets, OPenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) datasets generated using [Kids First DRC Consensus Calling Workflow](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc-consensus-calling.md) to calculate Tumor Mutation Burden (TMB). 
TMB is calculated for tumor samples in each experimental strategy (WGS, WXS, and Targeted Sequencing) using SNV calls for all cohorts and cancer types evaluated in the project. 
The TMB calculation is adapted from [snv-callers module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) of the OpenPBTA-analyses, but uses nucleotide variants (SNVs) in the consensus file called in at least 2 of the 4 caller (Mutect2, Strelka2, Lancet, and Vardict) utilized in the consensus calling workflow and/or 1 of the 4 callers if the SNV was a hotspot mutation. 
The SNV consensus is split into two subsets of SNVs and multinucleotide variants (MNVs). 
The MNVs subset is split into SNVs, merged to the SNVs subset, and sample-specific redundant calls removed. 
The resulting merged and non-redundant SNV consensus calls togther with sample-specific BED files are utilized for TMB calculation as described below. 
```

one sentence per line please.

I do not think we are doing this for targeted sequencing, are we?